DxDiagram improve Expr descriptions (#620)
DevExpress/devextreme-documentation: api-reference/10 UI Widgets/dxDiagram/1 Configuration/nodes/childrenExpr.md

The current node's data object.

---

> This option is in effect for nodes of the `"verticalContainer"` or `"horizontalContainer"` type.

Reviewer: ```suggestion
This option is in effect for `"verticalContainer"` or `"horizontalContainer"` nodes.
```

> As an alternative, use the [containerKeyExpr](/Documentation/ApiReference/UI_Widgets/dxDiagram/Configuration/nodes/#containerKeyExpr) option to provide a container's content.

Reviewer: ```suggestion
You can also use the [containerKeyExpr](/Documentation/ApiReference/UI_Widgets/dxDiagram/Configuration/nodes/#containerKeyExpr) option to provide a container's content.
```
